src/pybind/mgr/dashboard/tests/test_access_control.py

```diff
@@ -556,6 +557,17 @@ def test_set_user_password(self):
                                       'admin@user.com')
         self.assertGreaterEqual(user['lastUpdate'], user_orig['lastUpdate'])
 
+    def test_unicode_password(self):
+        self.test_create_user()
+        password = '章鱼不是密码'
+        with tempfile.TemporaryFile(mode='w+') as pwd_file:
+            pwd_file.write(password)
+            pwd_file.seek(0)
```

Comment:
not quite sure why we need to write to a file and then read from it for its content. but if you are trying to create a more real-world use case, it sounds good to me.